spdx/parsers/tagvaluebuilders.py

```diff
@@ -523,9 +522,50 @@ def set_annotation_spdx_id(self, doc, spdx_id):
                 doc.annotations[-1].spdx_id = spdx_id
                 return True
             else:
-                raise CardinalityError('Annotation::SPDXREF')
+                raise CardinalityError("Annotation::SPDXREF")
         else:
-            raise OrderError('Annotation::SPDXREF')
+            raise OrderError("Annotation::SPDXREF")
+
+
+class RelationshipBuilder(object):
+    def __init__(self):
+        # FIXME: this state does not make sense
+        self.reset_relationship()
+
+    def reset_relationship(self):
+        """
+        Reset the builder's state to allow building new relationships.
+        """
+        # FIXME: this state does not make sense
+        self.relationship_comment_set = False
+
+    def add_relationship(self, doc, relationship):
+        """
+        Raise SPDXValueError if type is unknown.
+        """
+        self.reset_relationship()
+        doc.add_relationship(relationship.Relationship(relationship=relationship))
```

Comment:
Here, the `relationship` module object is shadowed by `relationship` local variable.

Reply:
thank you @ShubhamKJha, I'm looking into it. I think I might have to change str 'relationship' everywhere !!